src/wasm/kotlin/gradle/unpack-kotlin-native-tools.gradle

```diff
@@ -0,0 +1,9 @@
+
+task unpackKotlinNativeTools(type: Copy) {
+//    from(configurations.kotlinNative.singleFile)
```

Comment:
Is here and below commented out intentionally? Add a comment to explain?

Reply:
Yeah, this is copied from the other Kotlin project. These tools aren't needed here, I'll remove.